src/main/java/com/epam/task6/logic/sort/AuthorSort.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.task6.logic.sort;
+
+import com.epam.task6.logic.comparator.AuthorComparator;
+import com.epam.task6.model.Book;
+
+import java.util.*;
+
+public class AuthorSort implements SortBook {
+
+    @Override
+    public List<Book> sortBook(List<Book> books) {
+        Comparator<Book> comparator = new AuthorComparator();
```

Comment:
I fact, you don't need multiple search implementation, comparators and a factory is enough, and then call Collections.sort() inside dao, otherwise it leads to duplication